steward/src/commands/keys/eth/import_from_key.rs

```diff
@@ -0,0 +1,61 @@
+use super::show::ShowKeyCmd;
+use crate::application::APP;
+use abscissa_core::{clap::Parser, Application, Command, Runnable};
+use k256::pkcs8::ToPrivateKey;
+use signatory::FsKeyStore;
+use std::path;
+
+#[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Import an external Eth key via private key.\n This command will recover a Eth key, storing it in the keystore. \n It takes a keyname and private key."
```

Comment:
We may want to specify the expected format of the private key, i.e. the "0x0000..." format and not a path to a private key file or something.

Reply:
Done!